contrib/checkstyle/src/mill/contrib/checkstyle/Checkstyle.scala

```diff
@@ -0,0 +1,18 @@
+package mill.contrib.checkstyle
+
+import mill.api.PathRef
+
+/**
+ * [[CheckstyleModule]] output.
+ *
+ * @param errors number of errors found
+ * @param report Checkstyle report
+ * @param transformations transformations applied on the Checkstyle report
+ */
+case class Checkstyle(errors: Int, report: PathRef, transformations: Seq[PathRef])
```

Comment:
Maybe call this `CheckstyleOutput`? Just to be a bit more explicit that it's the output of checkstyle, and not the interface or implementation or something else